Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -534,7 +541,14 @@ extension AVFoundationPlayback {
     }
 
     open override var supportDVR: Bool {
-        guard let duration = seekableTimeRanges.first?.timeRangeValue.duration.seconds else { return false }
-        return playbackType == .live && duration >= minDvrSize
+        guard playbackType == .live else { return false }
+
+        for range in seekableTimeRanges {
```

Comment:
I think we can inline this _for_ using something like this:

`return seekableTimeRanges.first(where: { $0.timeRangeValue. duration.seconds > minDvrSize})  != nil`